.github/workflows/main.yml

```diff
@@ -165,12 +165,8 @@ jobs:
       - name: late setup & validation, ch-image
         if: ${{ matrix.builder == 'ch-image' }}
         run: |
-          [[ $(bin/ch-image python-path) = /usr/local/bin/python3 ]]
-
-      - name: late setup & validation, all
-        run: |
-          bin/ch-test --is-pedantic all
-          bin/ch-test --is-sudo all
```

Comment:
Why remove these lines?

Reply:
I am not sure if I removed these for some reason, but it looks like I removed lines I didn't intend to.